drivers/lora/CMakeLists.txt

```diff
@@ -1,5 +1,9 @@
 # SPDX-License-Identifier: Apache-2.0
 
-zephyr_library_named(loramac-node)
+if(TARGET loramac-node)
```

Comment:
An explanation in a comment above is in order. E.g. something like:

> sx1276.c is added to the loramac-node library when it exists because it needs the same include directories as the loramac-node has.
> 
> When it does not exist a library of the same name is created because ...

Reply:
Done